packages/manager/modules/pci/src/projects/project/storages/databases/add/add.controller.js

```diff
@@ -164,15 +127,30 @@ export default class {
 
   onNetworkTypeChange(usePrivateNetwork) {
     if (!usePrivateNetwork) this.model.subnet = null;
+    this.getOrderData();
+  }
+
+  // bug in oui-numeric: on-change event is fired before model is updated
+  onNodeNumberChange() {
+    this.$timeout(() => this.getOrderData());
+  }
+
+  getNodesSpecTranslation() {
+    const prefix = 'pci_database_add_spec_nodes';
+    const range =
+      this.model.plan.nodesCount === this.model.plan.maxNodes ? '' : '_range';
+    const single = this.model.plan.nodesCount === 1 ? '_single' : '';
+    const translateKey = `${prefix}${range}${single}`;
+    return this.$translate.instant(translateKey, {
+      min: this.model.plan.nodesCount,
+      max: this.model.plan.maxNodes,
+    });
   }
 
-  prepareOrderData() {
+  getOrderData() {
+    this.orderAPIUrl = `POST /cloud/project/${this.projectId}/database/${this.model.engine.name}`;
```

Comment:
`POST` keyword expected ?
```suggestion
    this.orderAPIUrl = `/cloud/project/${this.projectId}/database/${this.model.engine.name}`;
```

Reply:
The `POST` is expected here, we use it to show the equivalent API call.